ocs_ci/helpers/helpers.py

```diff
@@ -1392,6 +1392,55 @@ def measure_pvc_creation_time_bulk(interface, pvc_name_list, wait_time=60):
     return pvc_dict
 
 
+def measure_multiple_pvc_deletion_time(interface, pvc_name, status):
+    """
+    Get the starting/ending deletion time of a PVC based on provisioner logs
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_name (str / list): Name of the PVC(s) for creation time
```

Comment:
Better rename it to pvc_name_list or pvc_list

Reply:
this can be string or list, prefer to keep it similar to other variable names